BootloaderCommonPkg/Library/TpmLib/TpmEventLog.c

```diff
@@ -382,4 +382,51 @@ TpmLogLocalityEvent (
   }
 
   TpmLogEvent ( &PcrEventHdr, (const UINT8 *)&StartupLocalityEvent);
+}
+
+/**
+  Get the TCG log Lasa and LasaEven info.
+
+  @param[out] Tcg2Lasa  Pointer to the Log area start address .
+  @param[out] Tcg2LasaEven  Pointer to the Last event logged.
+
+**/
+VOID
+GetTCGLogInfo (
+  OUT  UINT32  *Tcg2Lasa,
+  OUT  UINT32  *Tcg2LasaEven
```

Comment:
Should we use Tcg2LasaLastEvent instead according to the function comments ?

Reply:
Yeah.. I could change to Tcg2LasaLastEvent instead of Tcg2LasaEven.